### PR TITLE
Release notes 6.34

### PR DESCRIPTION
**Pull request by vkostyukov**

### CODE REVIEW

File: source/blog/2016-03-11-release-notes-6-34.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+With that said, we highly recommend upgrading from 6.33 to 6.34.
+
+### Highlights
+
+* Finagle now includes downstream/upstream addresses in its exceptions (see [RB 797082][rb797082])
+* [Response classifiers][repreq] are now enabled for HTTP servers (see [RB 800179][rb800179])
+* Two new histograms `request_payload_bytes` and `response_payload_bytes` are now reported for
+  HTTP, Mux, Thrift, and ThriftMux protocols (see [RB 797821][rb797821])
+* A number of deprecated methods was removed from the Finagle API (see [RB 800974][rb800974])
```
Comment (dschobel) [inline source/blog/2016-03-11-release-notes-6-34.md]: *were removed

File: source/blog/2016-03-11-release-notes-6-34.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* Stack Overflow bug was fixed in `AsyncStream.concat` (see [RB 804408][rb804408])
+
+### Further Plans
+
+We're actively working on migrating Finagle to Netty 4 (`finagle-core` is mostly done,
+`finagle-http` is our next target) and as part of this work we're planing to completely
+replace [codecs][codecs], which are pretty tight to Netty 3, with [stacks][stacks]. While this is
```
Comment (dschobel) [inline source/blog/2016-03-11-release-notes-6-34.md]: s/tight/tied/

**Comment by dschobel** [comment]
:ship:

File: source/blog/2016-03-11-release-notes-6-34.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+```
+com.twitter.finagle.mux.ServerError: Duplicate tag 6
+```
+
+With that said, we highly recommend upgrading from 6.33 to 6.34.
+
+### Highlights
+
+* Finagle now includes downstream/upstream addresses in its exceptions (see [RB 797082][rb797082])
```
Comment (kevinoliver) [inline source/blog/2016-03-11-release-notes-6-34.md]: what do you think of instead linking to the commit?

File: source/blog/2016-03-11-release-notes-6-34.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* [Response classifiers][repreq] are now enabled for HTTP servers (see [RB 800179][rb800179])
+* Two new histograms `request_payload_bytes` and `response_payload_bytes` are now reported for
+  HTTP, Mux, Thrift, and ThriftMux protocols (see [RB 797821][rb797821])
+* A number of deprecated methods was removed from the Finagle API (see [RB 800974][rb800974])
+* Stack Overflow bug was fixed in `AsyncStream.concat` (see [RB 804408][rb804408])
```
Comment (kevinoliver) [inline source/blog/2016-03-11-release-notes-6-34.md]: s/Overflow/overflow/

File: source/blog/2016-03-11-release-notes-6-34.md (hunk 3 above)
Comment (vkostyukov) [inline source/blog/2016-03-11-release-notes-6-34.md]: Right, it looks confusing. I use `RB_` labels but link them to Github commits. Didn't figure out what else we can use for labels.

Comment (kevinoliver) [inline source/blog/2016-03-11-release-notes-6-34.md]: oh you did! hm, maybe instead of using RB XXX we can use the first 8 bytes of the sha for the link?

**Comment by kevinoliver** [comment]
looks great.

File: source/blog/2016-03-11-release-notes-6-34.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+a pretty wide internal change, it should not affect Finagle users.
+
+### Changelog
+
+* [Finagle 6.34][finagle]
+* [Util 6.33][util]
+* [Scrooge 4.6][scrooge]
+* [TwitterServer 1.19][ts]
+* [Ostrich 9.17][ostrich]
+
```
Comment (kevinoliver) [inline source/blog/2016-03-11-release-notes-6-34.md]: how hard would it be to add a list of contributors and counts?

Comment (vkostyukov) [inline source/blog/2016-03-11-release-notes-6-34.md]: Do you mean this?

```scala
[tw-mbp-vkostyukov finagle (master)]$ git shortlog -sn --no-merges finagle-6.33.0..finagle-6.34.0
    12  Kevin Oliver
     6  Daniel Schobel
     6  Eugene Ma
     5  Vladimir Kostyukov
     4  Ruben Oanta
     2  Christopher Coco
     2  Jillian Crossley
     2  Eitan Adler
     1  Ryan Greenberg
     1  Stu Hood
     1  Antoine Tollenaere
     1  Yoshimasa Niwa
     1  Bing Wei
     1  Edward Samson
     1  Liam Stewart
     1  Lucas Langer
     1  Miguel Cervera
     1  Moses Nakamura
     1  Peter Schuller
```

**Pull request merged**